src/Microsoft.VisualStudio.ProjectSystem.Managed/xlf/Resources.es.xlf

```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="DataFlowFaults">
         <source>Project system data flow '{0}' closed because of an exception: {1}.</source>
-        <target state="new">Project system data flow '{0}' closed because of an exception: {1}.</target>
+        <target state="translated">Fujo de datos del sistema de proyectos "{0}" cerrado debido a una excepción: {1}</target>
```

Comment:
Typo: `Flujo`

Reply:
thank you for this feedback. Issue fixed